vitepress/docs/src/pages/software_guide/eyetrackvr_app_guide.md

```diff
@@ -85,15 +85,15 @@ Press the `Save and Restart Tracking` button.
 ### Wired configuration (experimental)
 #### Note this is a Beta app feature only. Please use the latest Open Beta in the Discord.
 
-Find your ESPs in Device Manager and figure out their COM port numbers i. e. ``COM4``.
+Find your ESPs in Device Manager and figure out their COM port numbers i.e., ``COM4``.
```

Comment:
`i.e.` ("id est", "that is") can be thought of as "in other words" or "**i**n **e**ssence", whereas `e.g.` ("exempli gratia") is "for example", or "**e**xample **g**iven". Regardless, comma goes before `e.g.` and `i.e.`.

```suggestion
Find your ESPs in Device Manager and figure out their COM port numbers, e.g. ``COM4``.
```